src/runtime/virtcontainers/kata_agent.go

```diff
@@ -1548,6 +1549,31 @@ func (k *kataAgent) handleLocalStorage(mounts []specs.Mount, sandboxID string, r
 	return localStorages, nil
 }
 
+// Add the source block type to DriverOptions in the volume with dm-verity
+func handleDmVerityBlockVolume(driverType, source string, verityInfo *types.DmVerityInfo, vol *grpc.Storage) (*grpc.Storage, error) {
+	no, err := json.Marshal(verityInfo)
```

Comment:
This looks a little confusing to me (implies that `no` might be a `bool`). How about:

```suggestion
	info, err := json.Marshal(verityInfo)
```

Reply:
Fixed.